libs/components/code-examples/src/lib/modules/filter-bar/filter-bar/basic/example.component.ts

```diff
@@ -31,17 +26,19 @@ export class FilterBarBasicExampleComponent {
     'staff-assigned',
     'entering-grade',
     'current-grade',
+    'application-fee-received',
   ]);
 
-  protected communityConnectionModal = CommunityConnectionFilterModalComponent;
-  protected currentGradeModal = CurrentGradeFilterModalComponent;
-  protected enteringGradeModal = EnteringGradeFilterModalComponent;
-  protected roleModal = RoleFilterModalComponent;
-  protected staffAssignedModal = StaffAssignedFilterModalComponent;
+  protected applicationFeeModal = ApplicationFeeFilterModalComponent;
 
-  protected onModalOpened(
-    args: SkyFilterItemModalOpenedArgs<FilterItems>,
-  ): void {
-    args.data = of({ items: FILTER_SELECTION_VALUES[args.filterId] });
+  protected onSearchAsync(args: SkyFilterItemLookupSearchAsyncArgs): void {
```

Comment:
Instead of putting both the modal and lookup filter items in the same code example, it would be more beneficial to the consumer to see two separate examples, one for the modal (with its tests and harnesses), and one for the lookup (with its tests and harnesses). This way, we can more effectively showcase the differences between the two setups.

"Basic" code example would become "Filter bar with modal filters"
Create a new code example for "Filter bar with lookup filters"